kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -217,6 +219,14 @@
         if (this.pageName === PageNames.EXAM) {
           return 'exam-page';
         }
+        if (
```

Comment:
For another pattern, search for `LearnerReports.includes`.

Basically, you can add

```javascript
const RecommendedPages = [
  RECOMMENDED_POPULAR,
  RECOMMENDED_NEXT_STEPS,
  RECOMMENDED_RESUME,
  RECOMMENDED_OVERVIEW,
];
```

to `constants` and then just check `RecommendedPages.includes(this.pageName)`.